msteams-platform/webhooks-and-connectors/how-to/connectors-using.md

```diff
@@ -1,19 +1,21 @@
 ---
-title: Sending messages to Connectors and Webhooks
+title: Send messages to Connectors and Webhooks
 description: Describes how to use Office 365 Connectors in Microsoft Teams
 ms.topic: how-to
 localization_priority: Normal
 keywords: teams o365 connector
 ---
 
-# Sending messages to connectors and webhooks
+# Send messages to connectors and webhooks
```

Comment:
Self review is not done. All the images in this module are very old. get the latest images.